moztelemetry/dataset.py

```diff
@@ -72,7 +97,7 @@ class Dataset:
         schema = ['submissionDate', 'docType', 'platform']
 
         records = Dataset(bucket, schema) \\
-            .select(
+                .select(
```

Comment:
I think these are accidental whitespacing changes from a formatter. I would use `git add -p` to manually inspect the changes that you want to make.

Reply:
Okay I will, thanks.